my_calculator.rb

```diff
@@ -0,0 +1,59 @@
+# Display intro to User.
+puts "Let's have some Calculator Fun!"
+puts "You will need to enter a math operation and two whole numbers."
+# Ask User for input.
+puts "Please enter an operator (name or symbol) -  ex: add(+), subtract(-), multiply(*), divide(/)"
+operator = gets.chomp.downcase
+
+
+# Check if User input is included in array. If not, ask again.
+until ["add", "+", "subtract", "-", "multiply", "*", "divide", "/"].include?(operator)
+    puts "You have entered an invalid operator. Re-enter an operator:"
+    operator = gets.chomp.downcase
+end
+program_operator = operator
+
+
+# Ask User for input for first number.
+puts "Please enter your first number."
+num1 = gets.chomp
+# Verify User input is an integer - if not, ask again.
+while (num1 != '0') && (num1.to_i.to_s != num1)
```

Comment:
Suggest you default to doublequotes for strings.

```suggestion
while (num1 != "0") && (num1.to_i.to_s != num1)
```